kitsune/sumo/static/less/products.less

```diff
@@ -14,6 +14,7 @@
   font-family: 'Open Sans','Droid Arabic Naskh','Mplus Fulah',sans-serif;
   position: absolute;
   right: 0;
+  text-align: center;
```

Comment:
I *think* this screwed up the download button. See http://cl.ly/image/3Z1S183d1412

left is localhost and right is prod.